### PR TITLE
Create Moodle guide

### PR DESCRIPTION
**Pull request by juliamrch**
## Checklist

- [ ] My PR is related to an opened issue : NO

**⚠️ If changes affect GitHub Action files**

- [ ] I have added tests to cover my changes : [link](url)

## Pull request type

Please check the type of change your PR introduces:

- [x] Documentation content changes
- [ ] Bugfix on the site
- [ ] Build related changes
- [ ] Other (please describe):
      
## Description

_A summary of the changes_

A new guide has been added to deploy Moodle on Clever Cloud, as well as: 

- A new card for Moodle in `/guides/_index.md`
- Additional ressources in `guides/moodle.md`
- An icon for Moodle

Changes can be seen here: 

- Guides index: http://clever-doc-review-moodle-guide-pr-136.cleverapps.io/guides/
- Moodle guide: http://clever-doc-review-moodle-guide-pr-136.cleverapps.io/guides/moodle/

### Why is this needed?

This has been asked by some of our users.

## Reviewes
_Who should review these changes?_ @cnivolle

### CODE REVIEW

File: content/guides/moodle.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### Set Up Domain
+
+Moodle needs an URL declared in variables to work properly. You can set it up in **Domains names**, from your PHP application menu. If you don't have a domain name yet, you can use a `cleverapp.io` subdomain provided by Clever Cloud for test purposes.
+
+Don't forget to update `URL="<your-url"` if you haven't yet.
+
+### Deploy
+
+Get the remote in your application menu > **Information** > ** Deployùment URL** and add it to Git with `git remote add clever <clever-remote-url>`. Then, push your code with `git push clever -u master`
```
Comment (cnivolle) [inline content/guides/moodle.md:105]: * "Deployùment" has its weird accent  
* And the asterisks does not render boldly because of a whitespace.

**Comment by cnivolle** [comment]
Made some comments.
Also, I think we can provide a template for Guides and Docs. 

Most of the titles are not the same (sometimes we have an overview, sometimes it's missing).

File: content/guides/moodle.md (hunk 1 above)
Comment (juliamrch) [reply content/guides/moodle.md]: oups, fixed.

**Comment by juliamrch** [comment]
I tried to add a featured image for Twitter cards following [the Hugo doc](https://gohugo.io/templates/internal/#twitter-cards) and it was a big fail (see [this test](https://x.com/JuliaMarchh/status/1745108207150707039?s=20)) so something tells me we'll need to check what the theme is doing about that and maybe fix it? And, last two commits are just added images and aren't important.

**Pull request merged**